native_client/ctcdecode/__init__.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import absolute_import, division, print_function
+
+from . import swigwrapper
+
+
+class Scorer(swigwrapper.Scorer):
+    """Wrapper for Scorer.
+
+    :param alpha: Parameter associated with language model. Don't use
+                  language model when alpha = 0.
+    :type alpha: float
+    :param beta: Parameter associated with word count. Don't use word
+                 count when beta = 0.
+    :type beta: float
+    :model_path: Path to load language model.
+    :trie_path: Path to trie file (optional).
```

Comment:
Did we kept `trie` being optional ? I think we removed that, so maybe update the doc ?

Reply:
It's still optional at the API level, just not on the clients. But I'll remove it anyway.